make/autoconf/jvm-features.m4

```diff
@@ -543,10 +557,6 @@ AC_DEFUN([JVM_FEATURES_VERIFY],
     AC_MSG_ERROR([Specified JVM feature 'jvmti' requires feature 'services' for variant '$variant'])
   fi
 
-  if JVM_FEATURES_IS_ACTIVE(management) && ! JVM_FEATURES_IS_ACTIVE(nmt); then
-    AC_MSG_ERROR([Specified JVM feature 'management' requires feature 'nmt' for variant '$variant'])
```

Comment:
Have you verified that `management` functions as before even without the `nmt` feature?

Reply:
I didn't run any testing, I just looked at the code and confirmed the build still works. It appears to be a Hotspot feature, and Hotspot isn't used. If I'm wrong, the nightly builds will catch it.
Looking at https://github.com/ibmruntimes/openj9-openjdk-jdk17/blob/openj9/make/hotspot/lib/JvmFeatures.gmk#L125 the files are not in the JCL and INCLUDE_NMT isn't used.

Reply:
I didn't see the feature checked anywhere else.

Reply:
I do see `#if INCLUDE_NMT` in several tests, e.g. `test/hotspot/gtest/nmt/test_nmt_buffer_overflow_detection.cpp`, but they seem to depend upon libraries in `<test_image>/hotspot` that won't exist for OpenJ9 builds, so perhaps it's safe to omit `nmt` like this.

Reply:
Right, afaik we only run hotspot serviceability testing.